### PR TITLE
fix: update PascalCase for device resource names (#138)

### PR DESCRIPTION
**Pull request by iain-anderson**
- Aligns code with device profile changes made in PR 111

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-camera-go/blob/master/.github/CONTRIBUTING.md

## What is the current behavior?

Supplied device profiles don't work, as the resource names have been changed in the profiles (in PR #111 ) but not in the code

## What is the new behavior?
Updated code to match profiles

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions

Changes are untested - service should be tested before approval

### CODE REVIEW

File: internal/driver/driver.go [hunk 1]
```diff
@@ -127,63 +127,63 @@ func (d *Driver) HandleReadCommands(deviceName string, protocols map[string]mode
...
+		case "OnvifNtp":
 			data, err = onvifClient.GetNTP()
 			if err != nil {
 				d.lc.Error(err.Error())
 				return responses, err
 			}
 
 			cv, err = sdkModel.NewCommandValue(reqs[i].DeviceResourceName, common.ValueTypeString, string(data))
-		case "onvif_system_reboot":
+		case "OnvifSystemReboot":
```
Comment (lenny-intel) [inline internal/driver/driver.go:170]: I didn't find this one defined in any of the profiles.

Comment (iain-anderson) [reply internal/driver/driver.go:170]: No. I've changed it anyway, for consistency with the other operations

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**